test/VsixTesting.Xunit.Tests/VsixTesting.Xunit.Tests.csproj

```diff
@@ -27,4 +27,13 @@
   <ItemGroup>
     <Reference Include="Microsoft.CSharp" />
   </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\src\VsixTesting.Invoker\VsixTesting.Invoker.csproj" ReferenceOutputAssembly="false" />
+    <ProjectReference Include="..\\..\src\VsixTesting.Invoker\VsixTesting.Invoker.csproj" ReferenceOutputAssembly="false" CopyVsix="true" ExtensionsDirectory="CopyVsix/True.Relative" />
```

Comment:
❓ Can you specify `CopyVsix` without specifying `ExtensionsDirectory`?

Reply:
Yes because an empty `ExtensionsDirectory` doesn't have a root so the final `ExtensionsDirectory` path will be `$(OutDir) + empty = $(OutDir)`

https://github.com/josetr/VsixTesting/blob/83712a3865826de8284a1097c23044da1705f53d/src/VsixTesting.Xunit/VsixTesting.Xunit.targets#L22